cpp/ql/src/experimental/Security/CWE/CWE-1041/FindWrapperFunctions.ql

```diff
@@ -0,0 +1,140 @@
+/**
+ * @name Find Wrapper Functions
```

Comment:
I think much of the _implementation_ of this query involves finding wrapper functions, but I think the intention of this query is to find missing calls to said wrapper functions. If you agree with this, how about this name instead+
```suggestion
 * @name Missed opportunity to call wrapper function
```